README.md

```diff
@@ -27,6 +28,38 @@ Rich IntelliSense (completions) for `Dockerfile` and `docker-compose.yml` files:
 
 ![IntelliSense for DockerFiles](images/intelliSense.gif)
 
+## Debugging (Preview)
+
+To debug a .NET Core web application running in a Linux Docker container, add a Docker .NET Core launch configuration:
```

Comment:
Would it be better to say .NET Core ASP.NET app?  That's what they're called when adding dockerfiles to the workspace.

Reply:
Since the "preview" targets only ASP.NET, makes sense.  Updated.